identity-service/src/txRelay.js

```diff
@@ -27,17 +26,17 @@ async function delay (ms) {
   return new Promise(resolve => setTimeout(resolve, ms))
 }
 
-async function getGasPrice (req, web3) {
+async function getGasPrice (web3) {
```

Comment:
the reason we plumbed req through here is so we print out request id for the whole lifecycle of the request. we could we pass in the appropriate logger object to this function so can still get that request id? 

like if you have a req, it'll pass in req.logger, otherwise just logger

Reply:
I see, ill add back the req object to preserve the request id!